Core/bin/README.md

```diff
@@ -0,0 +1,73 @@
+Scripts
+==========
+
+Scripts modules first of all loads the environment variables by the `getenv.sh` script. Some of the most important variables that are needed are:
+
+- MA_ENV_PROPERTIES
+- MA_ENV_PROPERTIES
+- mango_paths_data
+
+Mainly this 3 variables specify the core for mango, where the source code and libraries are; as well as the data base files. All of them are properly know if Mango was set up by an installer, whenever we run a start_mango.sh all required variables must be figured out by this `getenv.sh` if the installetion was done like that.
+
+Another important thing to consider is that we also have a `mango.properties` file where we can change different values; normally this file is populated with lots of variables commented and also it contains a description of each variable value.
+
+Also there are other variables that can be useful to declare in case of deployments of other actions, this variables are:
+
+- mango_paths_data
+- keystore
+- MA_KEYSTORE
+- MA_KEYSTORE_PASSWORD
+- MA_KEY_PASSWORD
+- MA_KEY_ALIAS
+
+It is important to know that if needed we can customized the value from this variables by setting the value in the `mango.properties` or even in the `env.properties`.
+
+Note: We need to make sure the values are properly defined as if not default values will be considered. This is part of an issue reported but really it is not an issue that default values are taken, we just need to make sure we are defining the right values are populated. That it is why additionally some messages are added so at some point the user was able to look at the environment values considered. 
+
+So when running a script it is basic to validate that `mango_paths_home` is correct, it must be the path where we set the Mango installation path. And also we need to check `mango_paths_data` that is where the data base files and sources will be allocated.
+
+The environment values should be printed something like this:
+
+
+
+`+++++ Mango configuration found on /Users/juan.garcia/mango.properties`
+
+`+++++ MA_HOME is`
```

Comment:
Remove references to MA_HOME, it should be `mango_paths_home` now.

Reply:
I removed references for MA_HOME and I refered mango_paths_home instead.